src/erlcloud_s3.erl

```diff
@@ -560,11 +560,11 @@ get_notification_filter(Path, Config) ->
 get_notification_filter_do([]) -> [];
 get_notification_filter_do(S3Key) ->
     #xmlElement{content = Content} = hd(S3Key),
-    [[{name, "Prefix"}, {value, Prefix}],
-     [{name, "Suffix"}, {value, Suffix}]] =
-    [erlcloud_xml:decode([{name, "Name", text}, {value, "Value", text}], C)
-     || C <- Content],
-    {filter, [{prefix, Prefix}, {suffix, Suffix}]}.
+    GV = fun(Key, PL) -> proplists:get_value(Key, PL) end,
```

Comment:
i've given a second look onto this and I question my self: why does a library filters out S3filters for a user without giving him any idea about that? 
tomorrow AWS can change filters to have `regex` or more possible values and the library will silently swallow them.

how about simplifing to not filter out `prefix`|`suffix` at all